src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -1311,6 +1315,14 @@ configurationRegistry.registerConfiguration({
 			type: 'string',
 			default: '',
 			tags: ['notebookLayout']
+		},
+		// --- Start Positron ---
+		[NotebookSetting.workingDirectory]: {
+			markdownDescription: nls.localize('notebook.workingDirectory', "Default working directory for notebook kernels. Supports variables like `${workspaceFolder}`. When empty, uses the notebook file's directory. Any change to this setting will apply to future opened notebooks."),
+			type: 'string',
+			default: '',
+			scope: ConfigurationScope.RESOURCE
```

Comment:
My only thought is it may get confusion when we're all-in on positron notebooks that this setting is in the original vscode notebooks contrib. I'm not sure we will respect _every_ one of the original vscode notebook config options. I _don't_ think it should go in the positron notebooks one though... So this is probably good for now and then maybe in the future we will split out to a new Switzerland positron(the editor)-specific-notebook-configs section?

Reply:
yeah, makes sense! We probably will still at least want to use the `notebooks.` prefix right? (I put the setting here because I thought it should start with that prefix since it only applies to notebooks)